Cargo.toml

```diff
@@ -440,6 +436,7 @@ tempfile = '3'
 test-case = '3'
 test-generator = '0.3'
 testing_logger = '0.1'
+tracing-test = "0.2"
```

Comment:
[**chore**](https://conventionalcomments.org/): Keep dependencies in alphabetical order